actions/ansible-docs-build-init/action.yml

```diff
@@ -36,6 +36,10 @@ inputs:
       The version of antsibull-docs to install. When set, it refers to a git ref from which to install.
       If not set, the latest version from PyPI is installed.
     required: false
+  provide-link-targets:
+    description: A comma separated list of link targets that should not cause reference errors. A small RST file will be created during the build which contains these labels.
```

Comment:
Discussed a bit in IRC, but the typical way of accepting multi-valued input for actions is is  via multi-line string. So in invocation it'd be:

```yaml
- uses: ....
  with:
    provide-link-targets: |
      target1
      target2
```

We don't necessarily have to do that; it's just a typical convention.

For example, see the use of `restore-keys` in the cache action:
https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#input-parameters-for-the-cache-action

Reply:
I rebased, and rewrote it to use newlines instead of commas.

(I'm a bit worried of using inputs in scripts though. If I'm writing `echo "${{ input.foo }}"`, and the user passes `"\nrm -rf /\necho "` as `foo`, would then the three commands `echo ""`, `rm -rf /`, `echo ""` be executed? I don't think this is a new problem with GHA in general though...)

Reply:
Possibly it could be mitigated by setting an env var first with a here-doc or something? Or maybe even with the `env:` block on the step (although I think that might be problematic). Though I don't know then if it's the same problem when using the variable. We could play with some possibilities. It'd also be fine to do this as a separate step if you want, in Python or whatever, if that's more comfortable.

The most important safety mechanism we have is that this will only ever run with a `contents: read` token, so the "damage" even if execution is possible, is probably limited to something like `exit 0` to ensure the step is successful when it shouldn't be.

Reply:
I think the only way to properly mitigate this is to set an env var with GHA itself. Everything else has the same drawbacks, or makes everything more complicated. I've implemented it in 13b1c0558072f943448b5eb5b7dac10fb083e475.